devtools/devtools/behave/temporarydirectory.py

```diff
@@ -5,11 +5,17 @@
 
 from pathlib import Path
 from tempfile import TemporaryDirectory
+import shutil
 
 
 def get_context_temp_dir_path(context) -> Path:
     if not hasattr(context, "temp_dir"):
         context.temp_dir = TemporaryDirectory()
+
+        # context.add_cleanup(
```

Comment:
`context.add_cleanup` has the behaviour of a stack, so moving it so that it's declared after the cleanup action which deletes the directory should ensure that it works.

Reply:
Literally no clue what you mean here.

Reply:
Let's talk on co-screen.